client/curriculum/html/Images.tsx

```diff
@@ -15,6 +15,7 @@ import Lesson, {
 import { Pava } from '~/services/contributors';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.